src/app/state/persist/types.ts

```diff
@@ -36,6 +36,7 @@ export interface PersistState {
   loading: boolean
   stringifiedEncryptionKey: undefined | 'skipped' | StringifiedKeyWithSalt
   enteredWrongPassword: boolean
+  passwordCheckPass: boolean
```

Comment:
Is there any other pattern we can use?
I think it's quite ugly that this is in state

and it's risky
- open tab1, export private key, get asked to type in password, ignore
- open tab2, use a future feature that requires passwordCheckPass (or export private key of another account)
- type password in tab2
- tab1 unlocks

Reply:
Not sure, I am open to suggestions how to respond to saga execution in UI differently.

I am missing some detail here. Actions are not synced across tabs why it would affect another tab?

Reply:
Oh, i wrote it down wrong, and there's only less serious issue
- open tab1, export private key, get asked to type in password, type pass, see private key
- open tab2, export private key for different account, see private key

Reply:
ahh thanks I can finally repro this :+1:

Reply:
I still don't like the pattern enough.
Could you merge this without validating password commits, and move those to a separate PR for later?

Reply:
additional stuff moved to https://github.com/oasisprotocol/oasis-wallet-web/compare/mz/privKeyPasswordCheck?expand=1